template.yaml

```diff
@@ -37,16 +45,24 @@ Resources:
       Runtime: python3.9
       Environment:
         Variables:
+          apiChartOfAccounts: '/accounts'
+          apiValidTags: '/tags'
+          CacheTTL: !Ref CacheTTL
           MipsOrg: !Ref MipsOrganization
           SsmPath: !Ref SsmParamPrefix
-          CacheTTL: !Ref CacheTTL
-          apiAllCostCenters: '/all/costcenters.json'
+          CodesToOmit: !Ref CodesToOmit
+          CodesToAdd: !Ref CodesToAdd
       Role: !GetAtt FunctionRole.Arn
       Events:
-        AllCostCenters:
+        ChartOfAccounts:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            Path: '/accounts'
```

Comment:
This `Path` property has to be a hard-coded string, a Ref is not allowed. And since it needs to be hard-coded here I went with hard-coding all of the occurrences in this file.

Reply:
ok